client/src/featureform/register.py

```
@@ -4876,6 +4886,137 @@
                 set_run("")
             clear_state()
 
+    DeletableResource = Union[
+        FeatureColumnResource,
+        SubscriptableTransformation,
+        LabelColumnResource,
+        TrainingSetVariant,
```

Ruff (0.8.2)

`TrainingSetVariant` may be undefined, or defined from star imports

(F405)